app/controllers/concerns/remote_record_controller.rb

```diff
@@ -0,0 +1,53 @@
+module RemoteRecordController
```

Comment:
Yes, I think it'll work either way.  We could definitely have a CurationConcernController and have ScannedResourcesController and MultiVolumeWorkController subclass it.  Does that seem better than having the shared behavior in a module?